lib/AST/RequirementMachine/RequirementMachineRequests.cpp

```diff
@@ -447,8 +447,12 @@ RequirementSignatureRequest::evaluate(Evaluator &evaluator,
       }
     }
 
+    // FIXME: We don't have the inverses from desugaring available here!
```

Comment:
StructuralRequirementsRequest could return a second list of inverses as part of the request result